Software/Development/SemesterProject/App/closedLoopControlManager.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * closedLoopControlManager.c
+ *
+ *  Created on: 25. apr. 2019
+ *      Author: Nicolai Fransen
+ */
+
+#include "closedLoopControlManager.h"
+
+void runClosedLoopControl(void)
+{
+    float movementReference = 0;
+    float iqReference = 0, idReference = 0;
+    float abcCurrents[2];
+    float theta = 0;
+    volatile dqObject dqCurrents, dqVoltages;

```

Comment:
volatile keyword was used only for testing (avoiding the compiler to optimize)

Reply:
Fixed